### PR TITLE
Adding WeeklyDevTips

### PR DESCRIPTION
**Pull request by ardalis**
Before raising a PR and it getting merged, make sure you have completed the following:

DONE:

### Things to do:

- [x] Add podcasts in appropriate categories in ascending order of podcast name
- [x] If there is no matching category, add a new category to the list in ascending order of category name, and add it to the table of contents in the same order
- [x] Add a brief description of the podcast
- [x] Add the frequency of episodes (Check the list for examples)

Thanks for your contributions and being awesome :) Cheers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ List of podcasts which are helpful for software engineers/programmers.
 
   - <b>Description</b>: show covering a broad range of .NET-related topics in considerable technical depth.
   - <b>Frequency</b>: Monthly once
+  
+- [Weekly Dev Tips](http://weeklydevtips.com)
```
Comment (rShetty) [inline README.md]: Is this part of .NET ?

**Comment by ardalis** [comment]
I didn’t see a general programming category so this was closest one. Should I create a code quality or improvement category?

**Comment by rShetty** [comment]
@ardalis There is a general category section

**Comment by ardalis** [comment]
Good point - moved.

**Pull request merged**